.gitignore

```diff
@@ -13,3 +13,5 @@ target/*
 .sdk/
 .sdk.tar.gz
 versions/*
+.idea/
```

Comment:
If we want to make a PR upstream, this is probably worth excluding—many maintainers like to keep editor-specific excludes in global gitignores instead of project-specific ones.

Reply:
Done.